wisp/wisp.php

```diff
@@ -170,6 +170,22 @@ function wisp_ConfigOptions() {
             "Type" => "text",
             "Size" => 25,
         ],
+        "additional_ports" => [
+            "FriendlyName" => "Additional Ports",
+            "Description" => "Additional ports in a JSON string, each must have a key of either "NONE" or an egg parameter name to set. E.g. If you want a game port, game port+1 and RCON_PORT assigned to game port +4 then use: {\"NONE\":1, \"RCON_PORT\":4} (optional)",
```

Comment:
It's probably easier to link to the README section explaining this with examples instead as this option will be confusing for most people.

Reply:
Agreed, I was thinking it gets a little awkward the longer those descriptions get. I've added a link to the readme with a much shorter description 👍